Library/Homebrew/download_strategy.rb

```diff
@@ -582,7 +601,9 @@ def fetch_repo(target, url, revision = nil, ignore_externals = false)
     svncommand = target.directory? ? "up" : "checkout"
     args = ["svn", svncommand]
     args << url unless target.directory?
-    args << target
+    target_arg = target.to_s
+    target_arg = escape(target_arg) if svncommand == "up"
+    args << target_arg
```

Comment:
Can you reorder this a bit? Currently we have to check twice for `target.directory?` and once for `svncommand == "up"`, which all check for the same thing.